tests/pyccel/test_pyccel.py

```diff
@@ -253,267 +253,266 @@ def teardown(path_dir = None):
+#                                         "scripts/import_syntax/from_mod_import_as.py",
+#                                         "scripts/import_syntax/import_mod.py",
+#                                         "scripts/import_syntax/import_mod_as.py",
+#                                         "scripts/import_syntax/from_mod_import_func.py",
+#                                         "scripts/import_syntax/from_mod_import_as_func.py",
+#                                         "scripts/import_syntax/import_mod_func.py",
+#                                         "scripts/import_syntax/import_mod_as_func.py",
+#                                         "scripts/import_syntax/collisions.py",
+#                                         "scripts/import_syntax/collisions3.py",
+#                                         "scripts/import_syntax/collisions5.py",
+#                                         "scripts/import_syntax/collisions6.py",
+#                                         ] )
+# def test_import_syntax( test_file ):
+#     pyccel_test(test_file)
+
+# #------------------------------------------------------------------------------
+# @pytest.mark.parametrize( "test_file", ["scripts/import_syntax/from_mod_import_as_user_func.py",
+#                                         "scripts/import_syntax/from_mod_import_as_user.py",
+#                                         "scripts/import_syntax/collisions2.py"
+#                                         ] )
+# def test_import_syntax_user_as( test_file ):
+#     pyccel_test(test_file, dependencies = "scripts/import_syntax/user_mod.py")
+
+# #------------------------------------------------------------------------------
+# @pytest.mark.parametrize( "test_file", ["scripts/import_syntax/from_mod_import_user.py",
+#                                         "scripts/import_syntax/import_mod_user.py",
+#                                         "scripts/import_syntax/import_mod_as_user.py",
+#                                         "scripts/import_syntax/from_mod_import_user_func.py",
+#                                         "scripts/import_syntax/import_mod_user_func.py",
+#                                         "scripts/import_syntax/import_mod_as_user_func.py",
+#                                         ] )
+# def test_import_syntax_user( test_file, language ):
+#     pyccel_test(test_file, dependencies = "scripts/import_syntax/user_mod.py", language = language)
+
+# #------------------------------------------------------------------------------
+# def test_import_collisions():
+#     pyccel_test("scripts/import_syntax/collisions4.py",
+#             dependencies = ["scripts/import_syntax/user_mod.py", "scripts/import_syntax/user_mod2.py"])
+
+# #------------------------------------------------------------------------------
+# def test_numpy_kernels_compile():
+#     cwd = get_abs_path(".")
+#     compile_pyccel(os.path.join(cwd, "scripts/numpy/"), "numpy_kernels.py")
```

Comment:
Were all these comments for debugging?
To run one particular test you can use the command:
```
pytest test_pyccel.py -k test_multiple_results
```

Reply:
I forgot about this temporary change.